### ISSUE TITLE
Get rid of pencil icons for Title and Description

### ISSUE THREAD

**Issue by waxlamp** [XDATA-Year-2/sitar#63]

**Comment by waxlamp** [comment]
The pencil icon is pretty ugly.  Trello (and I think other sites) handle things like title editing by just having the user click on the title.  (By comparison, GitHub uses a little "edit" button to the right of the title.)

If we go that route, we will need to have some kind of placeholder to click on for a blank description (since that is optional).

@jeffbaumes what are your thoughts on this?

**Comment by jeffbaumes** [comment]
Yes - the pencils are ugly :) A little Edit button/link sounds great. BTW red trash can also looks kinda ugly to me - I think it at least needs a larger icon/button there - perhaps the trash can with the word Delete.
